### DISCUSSION TITLE
Use brokenImagePath setting also for image transforms

### DISCUSSION THREAD

**Discussion by piotrpog** [craftcms/cms#10136] [Ideas] [answered]
I noticed that when we access url of asset that is missing directly, it is replaced by image set in `brokenImagePath`, but not when we access image transform of missing image.

**Comment by brandonkelly** [comment, marked as answer]
It should be used whenever Craft intercepts an image request (`Accept: image/*` header) that it doesn’t know how to handle.

If the image transform you are accessing is resolving to a valid file on the server, your web server is going to send it, regardless of whether the source image still exists (it doesn’t care).

If the file doesn’t exist, and you can see that the request has the `Accept: image/*` header, then perhaps your web server isn’t configured to pass the request off to Craft if the requested file doesn’t exist.
